src/structures/Guild.js

```diff
@@ -159,6 +159,8 @@ class Guild extends Base {
      * * VERIFIED
      * * VIP_REGIONS
      * * VANITY_URL
+     * * DISCOVERABLE
+     * * FEATURABLE 
```

Comment:
There's a trailing space, resulting on lint errors:
```suggestion
     * * FEATURABLE
```